aspnet/fundamentals/logging.rst

```diff
@@ -102,53 +139,36 @@ Note that in order to use formatting as well as logging the exception, we passed
-	:linenos:
-	:lines: 8-29
-	:dedent: 4
-	:emphasize-lines: 4,7,10
-
-By requesting an instance of ``ILogger<T>``, your class doesn't need to create an instance of a logger via ``ILoggerFactory``. You can use this approach anywhere you don't need the additional functionality offered by ``ILoggerFactory``.
-
 Configuring Logging in your Application
 ----------------------------------------
 
 To configure logging in your ASP.NET application, you should resolve ``ILoggerFactory`` in the ``Configure`` method in your ``Startup`` class. ASP.NET will automatically provide an instance of ``ILoggerFactory`` using :doc:`dependency-injection` when you add a parameter of this type to the ``Configure`` method. Once you've added ``ILoggerFactory`` as a parameter, you configure loggers within the ``Configure`` method by calling methods (or extension methods) on the logger factory. We have already seen an example of this configuration at the beginning of this article, when we added console logging by simply calling ``loggerFactory.AddConsole``. In addition to adding loggers, you can also control the verbosity of the application's logging by setting the ``MinimumLevel`` property on the logger factory. The default verbosity is ``Verbose``.
 
-.. note:: You can also specify the minimum level of logging to use on a per-logger basis as well. For example, the ``AddConsole`` extension method supports an optional parameter for setting its minimum ``LogLevel``.  
+.. note:: You can specify the minimum level of logging to use on a per-logger basis as well. For example, the ``AddConsole`` extension method supports an optional parameter for setting its minimum ``LogLevel``.  
 
 Configuring TraceSource Logging
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The built-in `TraceSourceLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta6/src/Microsoft.Framework.Logging.TraceSource/TraceSourceLogger.cs>`_ provides a simple way to configure log messages to use the existing `System.Diagnostics.TraceSource <https://msdn.microsoft.com/en-us/library/system.diagnostics.tracesource(v=vs.110).aspx>`_ libraries and providers, including easy access to the Windows event log. This proven means of routing messages to a variety of listeners is already in use by many organizations, and the ``TraceSourceLogger`` allows developers to continue leveraging this existing investment.
+The built-in `TraceSourceLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta7/src/Microsoft.Framework.Logging.TraceSource/TraceSourceLogger.cs>`_ provides a simple way to configure log messages to use the existing `System.Diagnostics.TraceSource <https://msdn.microsoft.com/en-us/library/system.diagnostics.tracesource(v=vs.110).aspx>`_ libraries and providers, including easy access to the Windows event log. This proven means of routing messages to a variety of listeners is already in use by many organizations, and the ``TraceSourceLogger`` allows developers to continue leveraging this existing investment.
```

Comment:
And a Debug logger that writes to System.Diagnostics.Debug.WriteLine so you can see logging output in the VS Output window: https://www.nuget.org/packages/Microsoft.Framework.Logging.Debug